crates/sui-indexer-alt/src/handlers/obj_info_pruner.rs

```diff
@@ -67,3 +94,8 @@ impl Handler for ObjInfoPruner {
         Ok(rows_deleted)
     }
 }
+
+impl FieldCount for ObjInfoToBePruned {
+    // This does not really matter since we are deleting, and we are not limited by the insertion limit.
```

Comment:
```suggestion
    // This does not really matter since we are not limited by postgres' bound variable limit, because
    // we don't bind parameters in the deletion statement.
```